test/remote/session_tests.js

```diff
@@ -5,15 +5,23 @@
 'use strict'
 
 const assert = require('insist')
-var TestServer = require('../test_server')
+const TestServer = require('../test_server')
 const Client = require('../client')()
-
-var config = require('../../config').getProperties()
+const P = require('../../lib/promise')
+const jwtool = require('fxa-jwtool')
+const config = require('../../config').getProperties()
+const pubSigKey = jwtool.JWK.fromFile(config.publicKeyFile)
+const duration = 1000 * 60 * 60 * 24
+const publicKey = {
+  'algorithm': 'RS',
+  'n': '4759385967235610503571494339196749614544606692567785790953934768202714280652973091341316862993582789079872007974809511698859885077002492642203267408776123',
+  'e': '65537'
+}
 
 describe('remote session', function() {
   this.timeout(15000)
   let server
-  config.signinConfirmation.skipForNewAccounts.enabled = true
+  config.signinConfirmation.skipForNewAccounts.enabled = false
```

Comment:
Having this enabled was making it hard to test for correct verification-state behaviour, so I disabled it and fixed the test below that was depending on it.